src/main/java/com/boomi/flow/services/boomi/mdh/common/EngineCompatibleDates.java

```diff
@@ -0,0 +1,15 @@
+package com.boomi.flow.services.boomi.mdh.common;
+import java.time.OffsetDateTime;
+import static java.time.format.DateTimeFormatter.ISO_OFFSET_DATE_TIME;
+
+public class EngineCompatibleDates {
+
+    public static String parse(OffsetDateTime offsetDateTime) {
```

Comment:
This method is called `parse` but you're calling `format` which does the opposite :thinking:

Reply:
thanks, changed now